pywinauto/timings.py

```diff
@@ -319,11 +319,45 @@ def WaitUntil(
             
     return func_val
 
+#=========================================================================
+def wait_until(
+    timeout, 
+    retry_interval, 
+    value = True, 
+    op = operator.eq):
+    
+    """
+	WaitUntil decorator 
+    """
+    
+    def wait_until_decorator(func):		
+        def wrapped(*args):

```

Comment:
It would be nice to have ```@wraps``` from _functools_ here to keep the original attributes
```python
from functools import wraps
def wait_until(..)
     def wait_until_decorator(func):	
         @wraps         
         def wrapped(*args):
```
More details at the end of this [article ](http://thecodeship.com/patterns/guide-to-python-function-decorators/)